include/Core/Drawable.hpp

```diff
@@ -22,6 +22,7 @@ class Drawable {
 public:
     virtual ~Drawable() = default;
     virtual void Draw(const Util::Transform &transform, const float zIndex) = 0;
+    virtual glm::vec2 GetOriginalSize() = 0;
```

Comment:
I believe this method should belong to Image and Text rather than being a part of Drawable. Subclasses of Drawable, such as hit-box, might not require this method.

Reply:
I think hit-box is not need to draw, so it is not drawable.
Or, if hit-box no have size, then the hit-box can't work.

Reply:
but  I fixed

Reply:
I think anything that is drawn on the screen should have `GetSize()`

Reply:
as for debugging hitbox, we could limit the number of hitbox of a `GameObject` to one,
and the `GameObject::Draw()` could be something like
```
void GameObject::Draw() {
    m_Drawable->Draw();
    if (debug && m_Hitbox != nullptr) {
       m_Hitbox->DrawOutline();
    }
}
```